### ISSUE TITLE
Public holidays should be recognised

### ISSUE THREAD

**Issue by l0x** [baires/shouldideploy#887]
This is a more generic version of the previously addressed issue https://github.com/baires/shouldideploy/issues/594

The timezone should be used to lookup a calendar of public holidays in that region, and it should not recommend deploying on days before public holidays!

**Comment by elydpg** [comment]
Agreed! Also, the work week varies by locale. In my locale, Christmas is a normal workday (as are Sundays) so there's no reason for me not to deploy today 🤷

**Comment by elydpg** [comment]
This issue should really be titled "Localize weekends and holidays"

**Comment by wallacetm** [comment]
+1 on this one! If this feature was already developed, it wouldn't stop production today in Brazil!

**Comment by baires** [comment]
https://date.nager.at/Api Seems a good option for this

**Comment by elydpg** [comment]
> https://date.nager.at/Api

Doesn't seem to localize weekends, though that can be done separately I think
